x-pack/platform/plugins/shared/alerting/server/alert_deletion/lib/test_utils.ts

```diff
@@ -7,87 +7,145 @@
 
 export const inactiveAlertsQuery = (days: number = 30, spaceId: string = 'space-1') => ({
   bool: {
-    should: [
+    filter: [
       {
         bool: {
-          filter: [
+          minimum_should_match: 1,
+          should: [
             {
               bool: {
-                should: [
+                filter: [
                   {
                     bool: {
-                      should: [{ match_phrase: { 'kibana.alert.workflow_status': 'closed' } }],
                       minimum_should_match: 1,
+                      should: [
+                        {
+                          bool: {
+                            minimum_should_match: 1,
+                            should: [
```

Comment:
Isn't a `should`, especially with `minimum_should_match: 1`, the same as a `must`?  Which wouldn't need the `minimum_should_match`.    And even if it should be `should`, I don't think we need the `minimum_should_match` since:

> If the bool query includes at least one should clause and no must or filter clauses, the default value is 1. Otherwise, the default value is 0. [(ref)](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html#bool-min-should-match)